include/cantera/zeroD/ConstPressureMoleReactor.h

```diff
@@ -0,0 +1,45 @@
+//! @file ConstPressureMoleReactor.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_CONSTPRESSMOLE_REACTOR_H
+#define CT_CONSTPRESSMOLE_REACTOR_H
+
+#include "cantera/zeroD/MoleReactor.h"
+
+namespace Cantera
+{
+
+/*!
+ * ConstPressureMoleReactor is a class for constant-pressure reactors
+ * which use a state of moles.
```

Comment:
```suggestion
 * ConstPressureMoleReactor is a class for constant-pressure reactors
 * which use a state of moles.
 * @since New in Cantera 3.0
```
Newly introduced objects should be marked (same applies for new methods)